src/cljs/rems/navbar.cljs

```diff
@@ -60,6 +60,14 @@
               text (get-in page [:translations language :title] (text :t/missing))]
           [nav-link url text])))))
 
+(defn logo []
+  [:div {:class "logo"}
+   [:div.img]])
```

Comment:
This is not the best place for the `logo` component (`navbar.cljs`), `atoms.cljs` would perhaps be better.

Reply:
agree